### PR TITLE
refactor(packages/sui-studio): Improve navigation of the header

### PR DESCRIPTION
**Pull request by turolopezsanabria**
Improve side panel navigation and header, especially for mobile

ISSUES CLOSED: #1490

## Description
The current navigation has a poor UX.
The side panel navigation and the header in Mobile were not optimal in SUI Studio.

* The button to show and hide the drawer is hidden on scroll
* The header is not fixed on top hence the user needs to scroll all the way up every time to change a component
* The font-size was too small
* The menu can't be collapsed on desktop
* etc

## Related Issue
[#1490](https://github.com/SUI-Components/sui/issues/1490)

## Before and after screenshots

**BEFORE**
before

**AFTER**

https://user-images.githubusercontent.com/23620759/194775906-132d7b5a-1bca-4771-a211-f447b96b1a4e.mov

https://user-images.githubusercontent.com/23620759/194775559-5cedaaf4-7b8c-4fbb-a11b-39aaa08a3f0f.mov

### CODE REVIEW

File: packages/sui-studio/src/components/layout/_style.scss [hunk 1]
```diff
@@ -11,43 +10,49 @@
   &-sidebar {
     @include breakpoint-from(s) {
       flex: 0 0 $w-sidebar;
-      transform: translate3d(0, 0, 0);
+      // transform: translate3d(0, 0, 0);
```
Comment (arnau-rius) [inline packages/sui-studio/src/components/layout/_style.scss:13]: Maybe we can remove it if it is no longer used.

**Comment by arnau-rius** [comment]
If you want to improve the burger button, and convert it to a close icon when the menu is open ➡️ https://www.youtube.com/shorts/ZsGUkdFJRxA (just pure css and html)

**Review by arnau-rius — APPROVED** [review]
Nice improvement! 👏🏽

**Comment by turolopezsanabria** [comment]
> If you want to improve the burger button, and convert it to a close icon when the menu is open ➡️ https://www.youtube.com/shorts/ZsGUkdFJRxA (just pure css and html)

oh!! top, I'll save it into my bookmarks and do it on a next PR

File: packages/sui-studio/src/components/layout/_style.scss [hunk 2]
```diff
@@ -11,43 +10,49 @@
...
-      display: block;
-      transform: translate3d(0, 0, 0);
+      transform: translate3d(-100%, 0, 0);
     }
   }
 
   &-main {
     background-color: $bgc-main;
+    position: relative;
+    top: $h-navHeader + 4;
```
Comment (jordevo) [inline packages/sui-studio/src/components/layout/_style.scss:41]: this same value `$h-navHeader + 4` is used at least twice so perhaps it's time to assign it its own variable?

**Review by jordevo — APPROVED** [review]

File: packages/sui-studio/src/components/layout/Logo.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import {getStudioLogo} from '../utils'
+import {getStudioLogo} from '@s-ui/studio/src/components/utils'
```
Comment (rmoralp) [inline packages/sui-studio/src/components/layout/Logo.js:1]: It could be better to use relative import instead, as it was

Comment (turolopezsanabria) [reply packages/sui-studio/src/components/layout/Logo.js:1]: yes! I agree 
don't know what happened here

File: packages/sui-studio/src/components/layout/_style.scss (hunk 1 above)
Comment (turolopezsanabria) [reply packages/sui-studio/src/components/layout/_style.scss]: ✅ done

File: packages/sui-studio/src/components/layout/Logo.js (hunk 3 above)
Comment (turolopezsanabria) [reply packages/sui-studio/src/components/layout/Logo.js]: ✅ done

File: packages/sui-studio/src/components/layout/_style.scss (hunk 2 above)
Comment (turolopezsanabria) [reply packages/sui-studio/src/components/layout/_style.scss]: ✅ done

**Review by rmoralp — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

**Pull request merged**